src/screens/Ticketing/Purchase/Overview/TicketDetailsSelection.tsx

```diff
@@ -0,0 +1,155 @@
+import MessageBox from '@atb/components/message-box';
+import FullScreenFooter from '@atb/components/screen-footer/full-footer';
+import {
+  PreassignedFareProduct,
+  PreassignedFareProductType,
+} from '@atb/reference-data/types';
+import {StyleSheet} from '@atb/theme';
+import {PurchaseOverviewTexts, useTranslation} from '@atb/translations';
+import MessageBoxTexts from '@atb/translations/components/MessageBox';
+import React, {useEffect, useState} from 'react';
+import {View} from 'react-native';
+import {TariffZoneWithMetadata} from '../TariffZones';
+import {UserProfileWithCount} from '../Travellers/use-user-count-state';
+import {getPurchaseFlow} from '../utils';
+import DurationSelection from './components/DurationSelection';
+import PurchaseMessages from './components/PurchaseMessages';
+import StartTimeSelection from './components/StartTimeSelection';
+import Summary from './components/Summary';
+import TravellerSelection from './components/TravellerSelection';
+import Zones from './components/Zones';
+import useOfferState from './use-offer-state';
+
+export type TicketDetailsSelectionProps = {
+  preassignedFareProduct: PreassignedFareProduct;
+  refreshOffer?: boolean;
+  selectableTravellers: UserProfileWithCount[];
+  selectableProductType?: PreassignedFareProductType;
+  fromTariffZone: TariffZoneWithMetadata;
+  toTariffZone: TariffZoneWithMetadata;
+  travelDate?: string;
```

Comment:
Is this and some other props redundant here?

Reply:
or maybe we want use it as default is user doesn't update it 🤔

Reply:
This _was_ true, but I redid all of this now 😅